app/controllers/application_controller.rb

```diff
@@ -4,9 +4,7 @@ class ApplicationController < ActionController::Base
   USE_SSO = Rails.configuration.sso['use_sso'].freeze
 
   def verify_sso_cookie(sf_object)
```

Comment:
@jpslav do you have an example of how I would implement the `CannotFindUserContact` exception from highlights?

Reply:
because doorkeeper is not part of this check, can't really change it to only verifying if a contact exists.
attempted to cleanup how this is working though and we can take a look in another PR (this one is TOO BIG! lol